UI/admin_signin.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Maintenance Tracker | Repairs | Online dignostics</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link 
+      rel="stylesheet" type="text/css" media="screen" href="styles/main.css" 
+    />
+</head>
+<body>
+  <header>
+    <h1>MaintenanceTracker</h1>
+    <p>
+      The Web Application for all your diagnostics, repairs, 
+      online consultations. Soutions for hardware, software &amp; 
+      firmaware issues.
+    </p>
+    <nav>
+      <a href="index.html">Home</a> |
+      <a href="#">User Sign In</a>
+      <a href="#">
+        <img src="images/user_avatar_sm-46px.png" alt="profile picture">
+      </a>
+    </nav>
+  </header>
+  <hr>
+  <main>
+    <div>
+      <h2>Notice Board</h2>
+      <p>
```

Comment:
Maybe add `<h4> Attn Admins<h4>`?

Reply:
Try that, see if it makes a good difference

Also for the PR title remove the [ ] around the PT story ID